feedback.md

```diff
@@ -0,0 +1,47 @@
+INFO 343 Final Project Proposal
+
+Describe what your team is proposing to build
+
+We propose to make a website with quizzes that generate a result based on the user's answers. User will be prompted with a multiple choice quiz embellished with pictures and an intuitive UI. After this, the quiz will spit out a result based on the answers to said question. Pre-rendered example quizzes will be provided for users to play around with at launch. We are also proposing to add a feature to this website which allows the user the tools to build their own Buzzfeed-esque quiz. Users should be able to save quizzes after creation. This, however, is a stretch goal.
```

Comment:
I've done something similar to this with questionnaires. Creating quizzes will be a breeze with a document based database like firebase. However, the real challenge, in my opinion, will be creating different types of quiz questions (free response, multiple choice, short answer, etc.).

Creating quizzes could be just a simple app, but I think you guys should allow users to share different quizzes online. It could be a list of quizzes online or even just a simple shareable link to access a single disposable quiz. Implementation of authentication to allow users to edit and manage their quiz would be great too.